docs/_posts/0000-01-03-continuous-delivery-and-deployment.md

```diff
@@ -178,21 +188,31 @@ Note:
 
 ## The case for branching…
 
+
+--
+
+## Branches
+
++ Protect the main code-base from incomplete changes
++ Enable parallel streams of development
++ Make the history more legible
+
 --
 
 ## Feature Branch Workflow
 
 > “… all feature development should take place in a dedicated branch instead of the master branch. … makes it easy for multiple developers to work on a particular feature without disturbing the main codebase. It also means the master branch will never contain broken code …”
 > —[Git Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow)
 
----
+--
 
 ## How do feature branches enable CD?
 
-* Nobody works on the master branch
-* Any feature based changes made on a feature branch
++ Nobody works on the master branch
++ Any feature-based changes made on a feature branch
++ Some Continuous Integration systems only run on master...
```

Comment:
Are you going to cover that this can mean practically that you are not practising continuous integration, or shall I include that in my argument?

Reply:
I'll put it in the notes to mention it